server/routes/activities/create-an-activity/handlers/startJourney.ts

```diff
@@ -1,11 +1,17 @@
 import { Request, Response } from 'express'
+import MetricsService from '../../../../services/metricsService'
+import MetricsEvent from '../../../../data/metricsEvent'
+import { initJourneyMetrics } from '../../../../utils/metricsUtils'
 
 export default class StartJourneyRoutes {
+  constructor(private readonly metricsService: MetricsService) {}
+
   GET = async (req: Request, res: Response): Promise<void> => {
     req.session.createJourney = {}
-    req.session.journeyMetrics = {
-      journeyStartTime: Date.now(),
-    }
+    initJourneyMetrics(req)
+    this.metricsService.trackEvent(
+      MetricsEvent.CREATE_ACTIVITY_JOURNEY_STARTED(res.locals.user).addJourneyStartedMetrics(req),
+    )
```

Comment:
How would we tie together the start and end of a single create or edit journey if the start and end are different events?

Reply:
Journey ids are how we're doing it on other journeys. The only way to tie events from a create journey at the moment would be by looking at the event times and usernames and making assumptions about which completed event belongs to which started event.

This is one of the reasons I'm keen to look at adding journey ids to create/edit.